### PR TITLE
ILMS-313 - fix console error - reset media recorders and blob files w…

### PR DESCRIPTION
**Pull request by Channa53**
…hen needed.

### PR Info
- [x] Please functional test this PR
- [ ] Already tested in KMS and stand alone

### CODE REVIEW

File: src/components/Recorder/Recorder.tsx [hunk 1]
```diff
@@ -48,14 +47,23 @@ export class Recorder extends Component<Props> {
...
         if (discard) {
             this.recordedBlobs = [];
             this.screenBlobs = [];
+            this.mediaRecorder = undefined;
+            this.screenRecorder = undefined;
             return;
         }
 
+        if (!videoStream) {
+            this.screenRecorder = undefined;
```
Comment (nireliyahu) [inline src/components/Recorder/Recorder.tsx:61]: @Channa53  are you sure you want here to reset the 
            this.screenRecorder = undefined;
 ?

Comment (Channa53) [reply src/components/Recorder/Recorder.tsx:61]: yes, it's neing used only while recording. and during that time no props or states should be updated

Comment (Channa53) [reply src/components/Recorder/Recorder.tsx:61]: ah you right I have an issue here..

Comment (Channa53) [reply src/components/Recorder/Recorder.tsx]: fixed

**Review by nireliyahu — APPROVED** [review]

**Review by nireliyahu — APPROVED** [review]

**Pull request merged**